drivers/osdp/Kconfig.pd

```diff
@@ -0,0 +1,217 @@
+#
+# Copyright (c) 2020 Siddharth Chandrasekaran <siddharth@embedjournal.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config OSDP_NUM_CONNECTED_PD
+	int
+	default 1
+	help
+	  In PD mode, number of connected PDs is is always 1 and cannot
+	  be configured.
+
+config OSDP_PD_ADDRESS
+	int "PD Address"
+	default 1
+	help
+	  The 7 least significant bits represent the address of the PD to which
+	  the message is directed, or the address of the PD sending the reply.
+	  Address 0x7F is reserved as a brodcast address to which all PDs would
+	  respond.
+
+menu "PD ID Information"
+
+config OSDP_PD_ID_VENDOR_CODE
+	hex "PD Vendor Code"
+	default 0x001A2B3C
+	help
+	  IEEE assigned OUI. Least 24 bits are valid.
+
+config OSDP_PD_ID_MODEL
+	int "PD Product Model Number"
+	default 1
+	help
+	  Manufacturer's model number. Least 8 bits are valid.
+
+config OSDP_PD_ID_VERSION
+	int "PD Product Version"
+	default 1
+	help
+	  Manufacturer's version of this product. Least 8 bits are valid.
+
+config OSDP_PD_ID_SERIAL_NUMBER
+	hex "PD Serial Number"
+	default 0xCAFEBABE
+	help
+	  A 4-byte serial number for the PD.
```

Comment:
If this is always 4 bytes, could you add range here so that user is not able to add invalid value.

Reply:
Done.